docs/contrib/env.md

```diff
@@ -4,6 +4,25 @@ Rubix is a Maven project and uses Java 8. It uses JUnit as the testing framework
 Ensure that you have a development environment that support the above 
 configuration.
 
+## Pre-requisites
+
+* `thrift` binary needs to be available at `/usr/local/bin/thrift`.
+  For Debian-based systems:
+  ```sh
+  sudo apt-get install thrift-compiler
```

Comment:
From what I remember, latest version of thrift doesn't work until we upgrade and I needed to install 0.9.6 specifically. @raunaqmorarka did you also see issues with latest version of thrift binary?

Reply:
I did not try the latest version of thrift binary. Current docker image for build contains thrift version 0.9.3 built from source. On a dev box we could use any version of thrift but for production we would want the same version of thrift on the deployment and build machine.

Reply:
I got the build to work with 0.9.1 too. I think @raunaqmorarka is correct that the versions should ideally match on build and deploy.

Reply:
I tried this with thrift 0.11.0, which is installed with apt-get by default. Build fails with it, exactly like in this case: https://www.eclipse.org/lists/sw360-dev/msg00021.html

So version is important and we need to mention one. I also use 0.9.3, we can mention installation steps for that which you can find in [Dockerfile](https://github.com/qubole/rubix/blob/master/Dockerfile)

Reply:
Sure, I'll update this to point the exact version and add a note regarding trying to match build and deploy versions.

Reply:
I opted to point to upstream docs since adding the steps for both Debian and CentOS here made the doc too long and mean that the package names and steps would have needed effort to make sure they stay in sync.

I added a note about taking care about thrift versions and the version Qubole mostly uses. Feel free to reword if something looks off.

Reply:
Changed to mention the steps to build from source which should be common across systems.